### PR TITLE
SIP Initial Recipe

### PR DESCRIPTION
**Pull request by miqlas**
Maybe we need further modification in /specs/haiku-g++. It is just the linux one, renamed.
Tested, it works, but i'm not sure about the flags.

### CODE REVIEW

File: dev-python/sip/patches/sip-4.17.patch [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+index 82140ac..ae39af3 100644
+--- a/configure.py
++++ b/configure.py
+@@ -152,7 +152,8 @@ def set_build_platform():
+         "sunos5":       "solaris-cc",
+         "ultrix":       "ultrix-g++",
+         "unix_sv":      "unixware-g++",
+-        "unixware":     "unixware-cc"
++        "unixware":     "unixware-cc",
++        "haiku1":		"haiku-g++"
```
Comment (waddlesplash) [inline dev-python/sip/patches/sip-4.17.patch]: "Haiku1"?

Comment (miqlas) [inline dev-python/sip/patches/sip-4.17.patch]: Yes, Python told me.

Comment (mmuman) [inline dev-python/sip/patches/sip-4.17.patch]: Yes python uses the OS name concatenated with the release number.

**Comment by waddlesplash** [comment]
Please squash your commits.

**Comment by miqlas** [comment]
I think it is in "good enough" shape now, so you can merge it.
I'm not happy with the hardcoded settings, but every other platform using that, so it is okay for now.

**Pull request merged**